tests/models/test_gpu.py

```diff
@@ -234,20 +212,21 @@ def test_parse_gpu_returns_none_when_no_devices_are_available(mocked_device_coun
         "LOCAL_WORLD_SIZE": "2",
     },
 )
+@mock.patch("torch.cuda.is_available", return_value=True)
 @mock.patch("torch.cuda.device_count", return_value=1)
```

Comment:
technically, these two need to be flipped in the order, since below the order is `_, mocked_device_count`

Reply:
good catch